### ISSUE TITLE
[1.0.11] Server crash when a no client's side mod is loaded

### ISSUE THREAD

**Issue by frandev94** [AllTheMods/ATM-8#928]
When startting the modmack server using [itzg/docker-minecraft-server](https://github.com/itzg/docker-minecraft-server) it tries to start the server with a client's side mod:

```
[20Feb2023 16:44:22.055] [main/ERROR] [net.minecraftforge.fml.loading.RuntimeDistCleaner/DISTXFORM]: Attempted to load class net/minecraft/client/KeyMapping for invalid dist DEDICATED_SERVER
[20Feb2023 16:44:22.056] [main/WARN] [mixin/]: Error loading class: net/minecraft/client/KeyMapping (java.lang.RuntimeException: Attempted to load class net/minecraft/client/KeyMapping for invalid dist DEDICATED_SERVER)
```

I think the error appears when tries to load _rubidium_extras-1.19.2_v1.3.2.jar_ (new on 1.0.11), a client side mod.
On 1.0.10 was working successfully.

Full debug file: [debug.log](https://github.com/AllTheMods/ATM-8/files/10786333/debug.log)
Compose example: [compose.yaml.txt](https://github.com/AllTheMods/ATM-8/files/10786492/compose.yaml.txt)

**Comment by Mitchell5200** [comment]
Looking through your Debug log, and checking the server Zip, it appears you are using a client pack for the server ,that client pack also appears to be missing 3 mods that are not meant to be server side anyway

**Comment by frandev94** [comment]
After changing the CF_FILE_ID to ___ now show up a new error:
```
2023-02-20 18:27:08 [init] Changing ownership of /data to 1000 ...
2023-02-20 18:27:08 [init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 2 1000 1000 4096 Feb 20 17:27 /data'
2023-02-20 18:27:10 [init] Resolved version given LATEST into 1.19.3 and major version 1.19
2023-02-20 18:27:10 [init] Resolving type given AUTO_CURSEFORGE
2023-02-20 18:27:12 [mc-image-helper] 17:27:12.430 INFO  : Processing modpack 'Server-Files-1.0.11.zip' (all-the-mods-8) @ 520914:4402354
2023-02-20 18:28:00 [init] ERROR failed to auto-install CurseForge modpack
2023-02-20 18:27:59 [mc-image-helper] 17:27:59.687 ERROR : 'install-curseforge' command failed. Version is 1.25.4
2023-02-20 18:27:59 me.itzg.helpers.errors.GenericException: Modpack is missing manifest.json
2023-02-20 18:27:59 at me.itzg.helpers.curseforge.CurseForgeInstaller.extractModpackManifest(CurseForgeInstaller.java:733)
2023-02-20 18:27:59 at me.itzg.helpers.curseforge.CurseForgeInstaller.processModpackZip(CurseForgeInstaller.java:299)
2023-02-20 18:27:59 at me.itzg.helpers.curseforge.CurseForgeInstaller.downloadAndProcessModpackZip(CurseForgeInstaller.java:285)
2023-02-20 18:27:59 at me.itzg.helpers.curseforge.CurseForgeInstaller.processModPack(CurseForgeInstaller.java:198)
2023-02-20 18:27:59 at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:135)
2023-02-20 18:27:59 at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:143)
2023-02-20 18:27:59 at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:21)
2023-02-20 18:27:59 at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
2023-02-20 18:27:59 at picocli.CommandLine.access$1500(CommandLine.java:148)
2023-02-20 18:27:59 at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
2023-02-20 18:27:59 at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
2023-02-20 18:27:59 at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
2023-02-20 18:27:59 at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
2023-02-20 18:27:59 at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
2023-02-20 18:27:59 at picocli.CommandLine.execute(CommandLine.java:2170)
2023-02-20 18:27:59 at me.itzg.helpers.McImageHelper.main(McImageHelper.java:123)
```
But it doesnt like to be referred to the actual issue. You can close this topic.

Thanks for the fast reply!

**Comment by Mitchell5200** [comment]
In that case it might just be coz it's a Zip folder of the pack.
But sure closing.
